perllib/Open311.pm

```diff
@@ -213,12 +218,12 @@ sub _generate_service_request_description {
 
 sub get_service_requests {
     my $self = shift;
-    my $report_ids = shift;
+    my $args = shift;
```

Comment:
The dates are being passed in here but never placed in params. Should params be a copy of args, apart from switching report_ids => service_request_id?